src/monmove.cpp

```diff
@@ -1570,3 +1576,69 @@ int monster::turns_to_reach( int x, int y )
 
     return static_cast<int>( turns + .9 ); // Halve (to get turns) and round up
 }
+
+void monster::shove_vehicle( const tripoint &remote_destination,
+                             const tripoint &nearby_destination )
+{
+    if( this->has_flag( MF_PUSH_VEH ) ) {
+        auto vp = g->m.veh_at( nearby_destination );
```

Comment:
```suggestion
        const auto vp = g->m.veh_at( nearby_destination );
```

Reply:
That won't allow changing vehicle properties or move it.